src/Assets/Lists/members.js

```diff
@@ -94,7 +94,7 @@ const memberData = {
                 {
                     fullName: "Zoey Xie",
                     position: "Logistics Associate",
-                    emailLink: "zoey.xie@mail.utoronto.ca",
+                    emaileLink: "zoey.xie@mail.utoronto.ca",
```

Comment:
Please fix this typo
```suggestion
                    emailLink: "zoey.xie@mail.utoronto.ca",
```